File: backend/src/tests/factories/message.py
from typing import Dict
from uuid import uuid4


def message_factory() -> Dict[str, str]:
    message = {
        "uuid": str(uuid4()),
        "title": "My other e2e title",
        "description": "My e2e description",
        "evenement_id": "event_id"
    }
    return message

File: front/src/app/interfaces/Message.ts

export interface Message {
  title: string;
  description: string;
  creator: {
    last_name: string;
    first_name: string;
    position: {
      group_id: string;
      group: {
        label: string;
      },
    },
    uuid: string;
  };
  created_at: string;
  uuid: string;
  tags: any[];
  resources: any[];
  evenement_id: string;
  type: string;
  type_label: string;
}